src/jsx-parser.ts

```diff
@@ -256,9 +266,12 @@ export class JSXParser extends Parser {
         }
 
         this.scanner.throwUnexpectedToken();
+
+        /* istanbul ignore next */
+        throw new Error('Unreachable');
```

Comment:
You could instead write `return `this.scanner.throwUnexpectedToken()`, which I assume returns `never`.